### PR TITLE
Working with React and Redux

### PR DESCRIPTION
**Pull request by robinbenitezmora**
In this project we are applying the concepts acquired in the last five weeks as React, Redux and how to work any application by means an API.

### CODE REVIEW

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @robinbenitezmora , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights :green_circle: 

- Fantastic commit history :heavy_check_mark: 
- No Linter error :heavy_check_mark: 
- Descriptive README.md file


## Suggested changes

- [ ] Hi @robinbenitezmora, great job on implementing this awesome weather app. However, I think we are not on the right path. In your project description, you are requested to select an API that provides **numeric data** about a topic that you like and then build the web app around it. 
> **Your web app will have several pages:**
  > - **Home page:**
     - When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
     - Along with the category name, you will display some numeric values per category that come from the API.
     - When the user clicks (or taps) on a category item, the application navigates to the details page.
  > - **Detail page**
    - n the details page, the web app retrieves data from the API to show detailed data in the category.
    - When the user clicks on the "Back" button (<), the user navigates to the home page.

Kindly :pray:  check once more the [project descriptions](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) and make the necessary changes

   .

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/spinner.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (tresorsawasawa) [inline src/components/spinner.js:1]: Home page:
- As mentioned by the previous reviewer kindly make sure When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
```
Comment (tresorsawasawa) [inline src/index.js:2]: - [ ] As mentioned by the previous reviewer kindly make sure When the user clicks on the "Back" button (<), the user navigates to the home page.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @robinbenitezmora 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**